index.html

```
@@ -437,7 +437,43 @@ <h2>⚛️ React Integration</h2>
 }</code>
       </div>
     </div>
+     <!-- Preact Integration -->
+    <div class="demo-section">
+      <h2>⚛️ Preact Integration</h2>
+      <p><strong>Step 1:</strong> Include CSS and JS via CDN</p>
+
+      <div class="code-block">
+  <button class="copy-btn" aria-label="Copy code">Copy</button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)